content/docs/contributing/release-process.md

```diff
@@ -97,6 +97,14 @@ First, ensure that you have all the tools required to perform a cert-manager rel
    with no scope ticked. It is used only by the `release-notes` CLI to
    avoid API rate limiting since it will go through all the PRs one by one.
 
+8. Ensure that you have `gsutil` installed; it's part of the Google Cloud SDK.
+
+9. Ensure you have `cosign` installed. See [the cosign installation instructions](https://docs.sigstore.dev/cosign/system_config/installation/).
+
+10. Ensure you have permissions to push to the OCI Helm chart registry, `quay.io/jetstack/charts`. You may need to run commands from [Helm's documentation](https://helm.sh/docs/topics/registries/#commands-for-working-with-registries).
+    Specifically, you will need to have run `helm registry login -u=USERNAME -p=PASSWORD quay.io` to authenticate with the registry.
+    Credentials can be obtained from the `quay.io` web UI if you're already a member of the `jetstack` organization on Quay.
```

Comment:
I can't log in to quay. I just get this error message.

<img width="960" height="510" alt="Image" src="https://github.com/user-attachments/assets/8ad42663-310b-41aa-bc4b-9d39641eba06" />